pkg/chunk/series_store.go

```diff
@@ -61,6 +61,11 @@ var (
 		// For 100k series for 7 week, could be 1.2m - 10*(8^(7-1)) = 2.6m.
 		Buckets: prometheus.ExponentialBuckets(10, 8, 7),
 	})
+	dedupedChunksTotal = promauto.NewCounter(prometheus.CounterOpts{
```

Comment:
This correlate with `cortex_chunk_store_stored_chunks_total` which is per user. @gouthamve are we OK having this metric non per-user?

Reply:
I don't know of a case where I would want to query this metric with a user dimension, that is, I'm not sure it's ever meaningful to know if one user has more de-duped chunks than another.

However if you were already querying `cortex_chunk_store_stored_chunks_total` with a user dimension and wanted to compare accurately with this metric then it would also need a user dimension.

My general feeling is there isn't a use case for checking de-duplication with a user and you would query both `cortex_chunk_store_stored_chunks_total` and this metric without a user label.

Reply:
You need it per-user if you're trying to charge users based on how much of the store they used.

Reply:
Is a count metric useful for this? Is there a good way to convert number of chunks flushed to the size of those chunks?

Reply:
It's all a little fuzzy since we don't include metadata in `cortex_ingester_chunk_stored_bytes_total` and don't have any user-specific counters for labels.

I'll modify #2353 to mention the bytes metric too.